README.md

```diff
@@ -52,3 +79,142 @@ struct small_FAB_Demo {
 }
 ```
 
+#### Medium FAB
+![medium_fab.png](images/medium.png)
+
+```
+import {FAB_regular} from '@ohos/Material_FAB'
+
+@Entry
+@Component
+struct medium_FAB_Demo {
+  build() {
+    Column(){
+      FAB_regular({
+        fabIcon : $r('app.media.add_icon'),
+        fabSize : 'medium',
+        position : {x : 100, y : 300},
+        backgroundColor : Color.Green
+      })
+    }.height('100%')
+  }
+}
+```
+
+#### Large FAB
+![large_fab.png](images/large_fab.png)
+
+```
+import {FAB_regular} from '@ohos/Material_FAB'
+
+@Entry
+@Component
+struct large_FAB_Demo {
+  build() {
+    Column(){
+      FAB_regular({
+        fabIcon : $r('app.media.add_icon'),
```

Comment:
Suggest to create as model for user inputs in the custom component.
Reference: https://github.com/Applib-OpenHarmony/Material_Ui_Checkbox/pull/1/files#diff-8ec4167e79402b1c41a8952ee93b5a535596d04dab9e2befccf8c3ab5c61b6dd